openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/DocumentRepository.java

```diff
@@ -82,7 +110,16 @@ public DocumentUpdater(Document original, Document updated, Operation operation)
     @Transaction
     @Override
     public void entitySpecificUpdate() {
-      recordChange("data", original.getData(), updated.getData(), true);
+      if (updated.getEntityType().equals(DefaultTemplateProvider.ENTITY_TYPE_EMAIL_TEMPLATE)) {
```

Comment:
what are we doing here if the entityType is not EMAIL will we not allow updating data? we just had to add validation in case of EMAIL rest should have been same